src/utility/ESCHideWrapper.jsx

```diff
@@ -0,0 +1,51 @@
+import react, {useEffect} from 'react';
+import { useStateContext } from "../contexts/ContextProvider";
+
+
+const ESCHideWrapper = (props) => {
```

Comment:
Better to get the destructured `component`, as there is no need to keep repeating with props.component again and again.